shim/src/realm.js

```diff
@@ -164,7 +164,7 @@ class BaseRealm {
       // note this would leak the parent TypeError, from which the child can
       // access .prototype and the parent's intrinsics, except that the Realm
       // facade catches all errors and translates them into local Error types
-      throw new TypeError('Realm only supports undefined or "inherited" intrinsics.');
+      throw new TypeError('Realm only supports undefined or "inherit" intrinsics.');
```

Comment:
this seems correct to me as-is, since both are adjectives

Reply:
I guess this value points the string value.

https://github.com/tc39/proposal-realms/blob/master/shim/src/realm.js#L128

Reply:
ah, then I’d backtick inherit, to show it’s not part of the English prose :-)

Reply:
that term was always confusing, and we should probably either change it, or change the way you signal the inheritance mechanism for intrinsics. But for now, I think we can stick to `inherit`.

Reply:
FWIW one of the PRs @erights and I will be filing today removes that option entirely in favor of direct `Realm.makeRootRealm()` and `Realm.makeCompartment()` calls. Modifying the new Realm is done with injected shims, since we found it's pretty hard to safely modify the intrinsics in-place or wrap them between realms.